---
layout: archive
title: ""
permalink: /software/
author_profile: true
---

<h1>Alignment and mapping</h1>

[strobealign](https://github.com/ksahlin/StrobeAlign) ultra-fast short-read alignment tool.

[StrobeMap](https://github.com/ksahlin/strobemers): finds all NAMs (local similar regions, roughly) between a set of query and reference sequences. 

<h1>Transcriptomics</h1>

[uLTRA](https://github.com/ksahlin/ultra): Aligns PacBio Iso-Seq or ONT cDNA/dRNA reads to a genome using a GTF annotation. uLTRA is shown to be highly accurate for short exons.

[isONcorrect](https://github.com/ksahlin/isONcorrect): Reference-free error correction of ONT cDNA and dRNA reads enabling reference free analysis of transcripts.

[isONclust](https://github.com/ksahlin/isONclust): Reference-free clustering of long ONT or PacBio Iso-Seq reads into genes.

[isONclust3](https://github.com/aljpetri/isONclust3): Reference-free clustering of long ONT or PacBio Iso-Seq reads into genes.

[isONform](https://github.com/aljpetri/isONform): Long-read transcript assembler.

[IsoCon](https://github.com/ksahlin/IsoCon): A tool for reconstructing transcripts from PacBio IsoSeq targeted sequencing reads. Its original use case was for primer captured cDNA from highly similar gene copies. However the methodology extends also other types of sequencing data with known start and stop sites, such as primer captured genomic data.


<h1>Genomics</h1>

[NGSpeciesID](https://github.com/ksahlin/NGSpeciesID): reference-free clustering and consensus forming of long-read amplicon sequencing.

[BESST](https://github.com/ksahlin/BESST): de novo genome assembly scaffolding using Illumina mate-pairs or paired-end reads.

[BESST_RNA](https://github.com/ksahlin/BESST_RNA): de novo genome assembly scaffolding using Illumina paired-end RNA-seq reads.


