---
layout: archive
title: "Software"
permalink: /software/
author_profile: true
---

<h2>Transcriptomics</h2>

[uLTRA](https://github.com/ksahlin/ultra): Aligns ONT cDNA and dRNA reads to a genome using a GTF annotation. Shown to be highly accurate for short exons.

[isONcorrect](https://github.com/ksahlin/isONcorrect): Reference-free error correction of ONT cDNA and dRNA reads enabling reference free analysis of transcripts.

[isONclust](https://github.com/ksahlin/isONclust): Reference-free clustering of long ONT or PacBio Iso-Seq reads into genes.

[IsoCon](https://github.com/ksahlin/IsoCon): A tool for reconstructing transcripts from PacBio IsoSeq targeted sequencing reads. Its original use case was for primer captured cDNA from highly similar gene copies. However the methodology extends also other types of sequencing data with known start and stop sites, such as primer captured genomic data.


<h2>Genomics</h2>

[NGSpeciesID](https://github.com/ksahlin/NGSpeciesID) is a software for reference-free clustering and consensus forming of long-read amplicon sequencing.

[BESST](https://github.com/ksahlin/BESST) is a software to perform de novo genome assembly scaffolding using Illumina mate-pairs or paired-end reads.

[BESST_RNA](https://github.com/ksahlin/BESST_RNA) is a software to perform de novo genome assembly scaffolding using Illumina mate-pairs or paired-end reads.


<h2>Alignment, mapping and sequence comparison</h2>

[StrobeMap](https://github.com/ksahlin/strobemers) is a software to find all NAMs (local similar regions, roughly) between a set of query and reference sequences. 

[StrobeAlign](https://github.com/ksahlin/StrobeAlign) is a software to map Illumina reads to a genome.